server/webhook.go

```diff
@@ -373,6 +373,12 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 		return
 	}
 
+	reopenedIssueMessage, err := renderTemplate("reopenedIssue", event)
```

Comment:
Outside the scope of this PR:
Why are we creating all the messages, if we are only going to use one? Wouldn't it make sense to check the action before we reach the for?

Reply:
Hi @larkox 
I realized that before adding third one. I though that's the way.

I can refactor this code. 
I would be glad to do if it is suitable for you

Reply:
I would like to hear the opinion of @hanzei before moving forward with that refactor.

Reply:
It totally makes sense to refactor the code :+1: Thanks for taking care of that @mustafayildirim

Reply:
I've refactored template rendering. @larkox @hanzei

Reply:
One more improvement on the refactor. Can we use the switch to decide just the template to use, and the rest of the repeated code outside of the switch (or in a function)?

Reply:
I fixed